.do/deploy.template.yaml

```
@@ -0,0 +1,9 @@
+spec:
+  name: docker-stellar-core-horizon
+  services:
+  - dockerfile_path: Dockerfile
+    git:
+      branch: acharb-pc-fork-quickdeploy
+      repo_clone_url: https://github.com/acharb/docker-stellar-core-horizon.git
```

We need to set these to this repo?
```suggestion
      branch: cap21and40
      repo_clone_url: https://github.com/leighmcculloch/stellar--docker-stellar-core-horizon.git
```

---

I can change to point to that. I was going to do after approved in case you wanted to test the button on my branch first (if I pointed to yours then the button wouldn't work bc not merged yet, annoyance of DO)